functions/go/inflate-helm-chart/metadata.yaml

```diff
@@ -0,0 +1,9 @@
+image: gcr.io/kpt-fn/inflate-helm-chart
```

Comment:
You can make verify-docs.py to skip your directory by add `inflate-helm-chart` in https://github.com/GoogleContainerTools/kpt-functions-catalog/blob/8e56c28a78baf04464520e5d6b3e29aa77382a03/scripts/verify-docs.py#L39

With that you don't need to add this file here.